src/oclif/command/BaseCommand.js

```diff
@@ -17,6 +17,16 @@ class BaseCommand extends Command {
   async init() {
     this.container = await createDIContainer();
 
+    // Set up config dir
+    /**
+     * @type {ensureHomeDir}
+     */
+    const ensureHomeDir = this.container.resolve('ensureHomeDir');
+    const homeDirPath = ensureHomeDir();
+    this.container.register({
+      configFilePath: asValue(homeDirPath + '/config.json'),
```

Comment:
This should be in the container since we already know `homeDirPath` there.

Reply:
I think I am not understanding awilix again here. Didn't you tell me [here](https://github.com/dashevo/mn-bootstrap/pull/123#pullrequestreview-470333267) to move this out of the container? We can't know the `homeDirPath` before running `ensureHomeDir()`, and we can't run the function before we get to init. So I don't know how we can register it on container creation, I think it need to be registered later? Also not clear how the user can specify a custom homedir here, where can we pass in and read the argument or env?